ouroboros-consensus-cardano/tools/db-analyser/Block/Alonzo.hs

```diff
@@ -0,0 +1,117 @@
+{-# LANGUAGE DeriveAnyClass             #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE NamedFieldPuns             #-}
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE TypeFamilies               #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Block.Alonzo (
```

Comment:
Would it make sense to refer to https://github.com/input-output-hk/cardano-ledger-specs/issues/2452 in the comments and explain that this module was copied from the node, why it was copied, and that it'll be removed from consensus once 2452 is solved?